command/pipelines/pipeline_save_test.go

```diff
@@ -46,6 +46,34 @@ func TestPipelineSave_basic(t *testing.T) {
 	}
 }
 
+func TestPipelineSave_stdin(t *testing.T) {
+	ts := GateServerSuccess()
+	defer ts.Close()
+
+	meta := command.ApiMeta{}
+	tempFile := tempPipelineFile(testPipelineJsonStr)
+	if tempFile == nil {
+		t.Fatal("Could not create temp pipeline file.")
+	}
+	defer os.Remove(tempFile.Name())
+
+	// Prepare Stdin for test reading.
+	tempFile.Seek(0, 0)
+	oldStdin := os.Stdin
+	defer func() { os.Stdin = oldStdin }()
+	os.Stdin = tempFile
+
+	args := []string{"--gate-endpoint", ts.URL}
+	fmt.Println(args)
```

Comment:
`*testing.T` has `Log` and `Logf` functions that silence output unless -v flag is passed or the test fails.  Do you intend for this to always be printed as part of test output?

Reply:
That shouldn't be there in the first place.